src/java.base/share/classes/java/util/Objects.java

```diff
@@ -261,15 +248,13 @@ public static <T> T requireNonNull(T obj, String message) {
     }
 
     /**
-     * Returns {@code true} if the provided reference is {@code null} otherwise
-     * returns {@code false}.
+     * {@return {@code true} if the provided reference is {@code
```

Comment:
See my previous comment on the "returns" verb in the doc comment for toString(Object, String). Here (IMO), you picked the better of the two options.